geoalchemy2/elements.py

```diff
@@ -1,6 +1,9 @@
+from __future__ import annotations
```

Comment:
Why is it needed? I don't see any use of this anywhere. Is it to support Py37?

Reply:
There's situations where the return type of methods are the same as the class type being defined. For example:

```python
class WKTElement(_SpatialElement):
    def as_wkt(self) -> WKTElement:
        if self.extended:
            srid_match = self._REMOVE_SRID.match(self.data)
            return WKTElement(srid_match.group(3), self.srid, extended=False)
        return WKTElement(self.data, self.srid, self.extended
```
This would cause a circular reference unless we use the newer feature that defers the evaluation of the type annotations until after the classes are defined. See https://peps.python.org/pep-0563/ for more details.

In Python 3.11 we could use the new `Self` annotation type: https://peps.python.org/pep-0673/, but the reason I didn't do it this way is because there's older versions of Python to support.

Reply:
Ah ok ok, thanks for the explanation. Let's keep it as is then.